tests/issue/109/unit/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.0)
+
+include("../../../cmake/modules/hunter_boost_component_b2_args.cmake")
+project(TestIssue109Unit)
+
+
+set(component "foo")
+set(cmakeArgs "FOO_OPT=1")
+
```

Comment:
It's designed to be:

```FOO_OPT_A=1;BOO_OPT_B=1``` => ```b2 -s OPT_A=1``` for component "foo" and ```b2 -s OPT_B=1``` for compoent "boo".

```FOO_OPT_A=1;FOO_OPT_B=1``` => ```b2 -s OPT_A=1 -s OPT_B=1``` for component "foo"

I'll check this with ```b2```'s help.